dired-async.el

```diff
@@ -81,6 +81,11 @@ or rename for `dired-async-skip-fast'."
   :risky t
   :type 'integer)
 
+(defcustom dired-async-password-regexp
```

Comment:
Can't we use `tramp-password-prompt-regexp`?

Reply:
This is a good idea. `tramp-password-prompt-regexp` seems to catch many more prompts, in many more languages.

I'm changing the PR accordingly.